engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -319,19 +304,16 @@ private void renderShadowMap() {
     public void renderWorldReflection() {
         PerformanceMonitor.startActivity("Render World (Reflection)");
 
-        DefaultRenderingProcess.getInstance().beginRenderReflectedScene();
-        GL11.glCullFace(GL11.GL_FRONT);
+        LwjglRenderingProcess.getInstance().beginRenderReflectedScene();
         playerCamera.setReflected(true);
 
-        playerCamera.lookThroughNormalized();
+        playerCamera.lookThroughNormalized(); // we don't want the reflected scene to be bobbing or moving with the player
         backdropRenderer.render(playerCamera);
         playerCamera.lookThrough();
 
         chunkShader.activateFeature(ShaderProgramFeature.FEATURE_USE_FORWARD_LIGHTING);
 
         if (renderingConfig.isReflectiveWater()) {
-            GL11.glEnable(GL_LIGHT0);
```

Comment:
From what I can see no light0 has been set anywhere, and by default an opengl light will be black, or making no contribution to the lighting of the scene. As such I've removed it, seemingly with no consequences.